src/Account/TransactionHistory/ChannelHistoryItem/ClaimDbets.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from 'react'
+import {
+    ListItem,
+    ListItemText,
+    Button,
+    List,
+    ListItemIcon
+} from '@material-ui/core'
+import SaveAltRoundedIcon from '@material-ui/icons/SaveAltRounded'
+
+const ClaimDebets: FunctionComponent<any> = props => {
```

Comment:
Typo

Reply:
fixed and added interface to the functional component